src/main/java/com/greedobank/cards/controller/CardTemplateController.java

```diff
@@ -0,0 +1,67 @@
+package com.greedobank.cards.controller;
+
+import com.greedobank.cards.dto.CardTemplateCreationDTO;
+import com.greedobank.cards.dto.CardTemplateCreationUpdateDTO;
+import com.greedobank.cards.dto.CardTemplateDTO;
+import com.greedobank.cards.service.CardTemplateService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@Validated
+@RequestMapping(path = "/api/v1/card-template")
+@Tag(name = "Card-template", description = "Interaction with card-template")
+public class CardTemplateController {
+    private final CardTemplateService cardTemplateService;
+
+    @Autowired
+    public CardTemplateController(CardTemplateService cardTemplateService) {
```

Comment:
we don't we use Lombok here, but in other classes
and moreover, `@Autowired` is redundant here

Reply:
Done